location-service/src/main/java/com/movie/recommender/location/exception/exceptions/LocationNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package com.movie.recommender.location.exception.exceptions;
+
+public class LocationNotFoundException extends RuntimeException {
```

Comment:
place this class and `RestResponseEntityExceptionHandler` under the exception package
there's no need for additional splitting into separate `exceptions` and `handlers` packages